packages/shared/src/components/auth/RegistrationForm.tsx

```diff
@@ -31,6 +32,10 @@ export type RegistrationFormValues = Omit<
   'method' | 'provider'
 >;
 
+interface EmailRegistrationForm extends RegistrationFormValues {
```

Comment:
Same here, can we not simply add it to the main one?

Reply:
I don't know why I did this. Lol. Let me refactor.

Reply:
The `RegistrationFormValues` is the object that we need to pass Kratos, the `EmailRegistrationForm` is just for the scope of this component.

Though I could probably just be overly typing the objects, I think we can add it to the original interface.